conf_files/huntsman.yaml

```diff
@@ -30,14 +30,11 @@ directories:
     resources: POCS/resources/
     targets: /var/huntsman/huntsman-pocs/conf_files
     webcam: webcams
-cameras:
```

Comment:
Removing the camera section from the default config file seems to have broken the tests. Don't think it should, though.